src/jquery.addressfield.js

```diff
@@ -179,6 +182,19 @@
   };
 
   /**
+  * Popualtes country dropdown with list of countries from provided json file if it is empty
+  */
+  $.fn.addressfield.initCountries = function(countryMap, selector){
```

Comment:
This also feels not-optimal. I feel like there is also a better way to pass the selector here.